Appz/Appz/Apps/Mail.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Mail.swift
+//  Appz
+//
+//  Created by Suraj Shirvankar on 12/5/15.
+//  Copyright © 2015 kitz. All rights reserved.
+//
+
+import Foundation
+
+
+public extension AvailableApplications {
+    
+    public struct Mail: ExternalApplication {
+        
+        public let scheme = "mailto:"
+        public let fallbackURL: String? = ""
+        
+        
+        public struct Email {
```

Comment:
I am comfortable with moving this to the global scope, for the sake of readability